Update dependencies in the project #877

ANUKULJHA1506 opened the pull request:
Updated all the dependencies, but the `extended_text_field` dependency needed to be downgraded because it required a higher SDK version than what was supported by the prescribed SDK levels.

ANUKULJHA1506 committed:
Updated All Dependencies

adityastic reviewed (commented):
Interesting. Are we talking about flutter sdk wrt to the downgrade or minimum android compilation sdk? I'm curious to know why 15 worked for us before

adityastic commented:
@ANUKULJHA1506 It would be much better if you can setup dependabot instead which updates dependencies automatically rather than us periodically updating them

ANUKULJHA1506 commented:
@adityastic  i m  working on dependabot ,And well yes in my android studio it was saying to have higher sdk version to use this as dependency as when i was doing pub get , it was constantly showing errors, thats why i lower the dependency version.

ANUKULJHA1506 committed:
Create dependabot.yml 

dependabot for updating packages on your own, at certain amount of time.

dependabot[bot] committed:
chore: bump flutter_lints from 3.0.2 to 4.0.0

Bumps [flutter_lints](https://github.com/flutter/packages/tree/main/packages) from 3.0.2 to 4.0.0.
- [Release notes](https://github.com/flutter/packages/releases)
- [Commits](https://github.com/flutter/packages/commits/flutter_lints-v4.0.0/packages)

---
updated-dependencies:
- dependency-name: flutter_lints
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore: bump extended_text_field from 14.0.0 to 16.0.0

Bumps [extended_text_field](https://github.com/fluttercandies/extended_text_field) from 14.0.0 to 16.0.0.
- [Release notes](https://github.com/fluttercandies/extended_text_field/releases)
- [Changelog](https://github.com/fluttercandies/extended_text_field/blob/master/CHANGELOG.md)
- [Commits](https://github.com/fluttercandies/extended_text_field/commits)

---
updated-dependencies:
- dependency-name: extended_text_field
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

ANUKULJHA1506 commented:
@adityastic ok , is their any slack or any other platform where i can connect with the fossasia community members.

ANUKULJHA1506 committed:
Update dependabot.yml

ANUKULJHA1506 committed:
Merge pull request #2 from ANUKULJHA1506/dependabot/pub/extended_text_field-16.0.0

chore: bump extended_text_field from 14.0.0 to 16.0.0

ANUKULJHA1506 committed:
Merge pull request #1 from ANUKULJHA1506/dependabot/pub/flutter_lints-4.0.0

chore: bump flutter_lints from 3.0.2 to 4.0.0

adityastic commented:
@ANUKULJHA1506 indeed, there is a gitter channel in the README. Also can you remove pubspec changes. We'll get dependabot to make them. Thanks!

adityastic commented on .github/dependabot.yml:
```diff
@@ -0,0 +1,32 @@
+    open-pull-requests-limit: 5  
+name: Auto-merge Dependabot PRs
+
+on:
```
Not sure why this is here. This is not dependabots config

ANUKULJHA1506 replied:
these will send automatically PRs to update the versions , upto 5 PRs can be send at a time, then you can check them and push it in main branch.

ANUKULJHA1506 replied:
do you want me to make some changes in depentabot config cause it is working fine in my forked repo.

adityastic replied:
There is a job configuration which is not relevant to dependabot.yml

adityastic replied:
I dont think it belongs here

adityastic commented on pubspec.lock:
```diff
@@ -167,10 +167,10 @@ packages:
       url: "https://pub.dev"
     source: hosted
-    version: "3.0.2"
+    version: "4.0.0"
```
We don't need changes in pubspec* files as dependabot will create PR's to update them. Can you remove both changes

ANUKULJHA1506 replied:
ok , I'm looking into it.

ANUKULJHA1506 committed:
Updated dependabot.yml

dependabot[bot] committed:
chore: bump flutter_blue_plus from 1.32.11 to 1.32.12

Bumps [flutter_blue_plus](https://github.com/boskokg/flutter_blue_plus) from 1.32.11 to 1.32.12.
- [Release notes](https://github.com/boskokg/flutter_blue_plus/releases)
- [Changelog](https://github.com/boskokg/flutter_blue_plus/blob/master/CHANGELOG.md)
- [Commits](https://github.com/boskokg/flutter_blue_plus/compare/1.32.11...1.32.12)

---
updated-dependencies:
- dependency-name: flutter_blue_plus
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

ANUKULJHA1506 committed:
Merge pull request #3 from ANUKULJHA1506/dependabot/pub/flutter_blue_plus-1.32.12

chore: bump flutter_blue_plus from 1.32.11 to 1.32.12

ANUKULJHA1506 commented:
@adityastic is dependabot.yml need some changes or it is fine i have removed the  two jobs that u asked for, now dependabot will create only PR's to update them.

adityastic commented:
@ANUKULJHA1506 both the pubspec file are still present

ANUKULJHA1506 commented:
@adityastic are you asking about dependabot.yml file or any other file ?

adityastic commented:
@ANUKULJHA1506 please read the last comment
